hops/numpy.py

```diff
@@ -0,0 +1,43 @@
+"""
+API for reading/writing numpy arrays to/from HopsFS.
+"""
+
+def load(hdfs_filename, **kwds):
+    """
+    Reads a file from HopsFS into a Numpy Array
+
+     Args:
+       :hdfs_filename: You can specify either a full hdfs pathname or a relative one (relative to your Project's path in HDFS).
+       :**kwds: You can add any additional args found in numpy.read(...) 
+
+     Returns:
+      A numpy array
+
+     Raises:
+      IOError: If the file does not exist.
+    """
+    import hops.hdfs as hdfs
```

Comment:
We can move imports to top of module unless any specific reason for importing inside the function. https://www.python.org/dev/peps/pep-0008/#imports